scripts/install

```diff
@@ -203,6 +204,8 @@ then
   ynh_permission_update --permission="main" --add="visitors"
 fi
 
+ynh_permission_create --permission="api" --url="/api" --allowed="visitors" --auth_header="false" --show_tile="false" --protected="true"
```

Comment:
Is it expected that the access to Mattermost API is allowed to visitors, even if the admin asked the app to be private?